apps/charterafrica/.env.template

```diff
@@ -13,6 +13,8 @@ S3_SECRET_ACCESS_KEY=
 S3_BUCKET=
 S3_REGION=
 
+CRONJOBS_SECRET_KEY=
```

Comment:
Name things based on function e.g. `RESOURCES_EDIT_KEY`, `RESOURCES_SECRET_TOKEN`, etc. We need this key to edit resources and not crons.